scripts/partition_manager/partition_manager.rst

```diff
@@ -96,34 +99,83 @@ placement: dict
          Place the partition after the first existing partition in the list.
 
      Valid values in the lists are ``app``, ``start``, ``end``, or the name of any partition.
+     The value ``start`` refers to the flash device's start address.
+     The value ``end`` refers to its end address.
      It is not possible to place the partition after ``end`` or before ``start``.
 
       align: dict
          Ensure alignment of start or end of partition by specifying a dict with a ``start`` or ``end`` key respectively, where the value is the number of bytes to align to.
          If necessary, empty partitions are inserted in front of or behind the partition to ensure that the alignment is correct.
          Only one key can be specified.
-         Partitions which directly or indirectly (through spans) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+         Partitions which directly or indirectly (through :ref:`spans <partition_manager_spans>`) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+
+   An empty value is also permitted, like so:
+
+   .. code-block:: yaml
+
+      mcuboot:
+        placement:
+
+   An empty value means that there are no constraints on the placement of the partition.
+
+span: list OR span: dict OR span: string
+   This property is used to define container partitions.
+   Its value may be a list, dict, or string.
+
+   Since this property is used to define container partitions, it cannot be used together with the ``placement`` property.
+
+   If the value is a list, its elements are the names of the partitions which should be placed in the container:
+
+   .. code-block:: yaml
+
+      # This partition spans, or contains, partition_1 through partition_n,
+      # in any order:
+      container_partition_name:
+        span: [partition_1, partition_2, ..., partition_n]
+
+   The list elements are interpreted as the set of potential partitions in the container, which Partition Manager may place on flash in any order. For example, ``partition_2`` could be placed before ``partition_1``.
```

Comment:
```suggestion
   The list elements are interpreted as the set of partitions in the container, which the Partition Manager may place in flash in any order. 
   For example, ``partition_2`` might be placed before ``partition_1``.
```

Reply:
I think the word "potential" is important.

This is not the exact set of partitions, because any partitions in the set which are not defined elsewhere are removed from the list before processing, as described by the next sentence.

Reply:
ok